api/global/trace_test.go

```diff
@@ -32,10 +32,10 @@ func (*testTraceProvider) Tracer(_ string, _ ...trace.TracerOption) trace.Tracer
 func TestMultipleGlobalTracerProvider(t *testing.T) {
 	p1 := testTraceProvider{}
```

Comment:
Should we change the name of this test struct to be `testTracerProvider` as well?